modules/monitoring-searching-and-filtering-metrics-targets.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-metrics-targets.adoc
+
+[id="monitoring-searching-and-filtering-metrics-targets_{context}"]
+= Searching and filtering metrics targets
+
+The list of metrics targets can be long. You can filter and search these targets based on various criteria.
+
+In the *Administrator* perspective, the *Metrics Targets* page provides details about targets for default {product-title} and user-defined projects.
+This page lists information for each target, including the service endpoint URL, the component being monitored, the status, the namespace, the last scrape time, and the duration of the last scrape.
```

Comment:
The list is totally fine as is, still easily readable. But if you make each item part of a bullet list, it could help scannability. Ex:

This page lists information for each target,  including the following:

- Service endpoint URL
- Component being monitored 
- Status 

So on and so forth.

Reply:
Good point. I think it's better as a bulleted list, and so I've made the change.